api/src/main/java/org/apache/cloudstack/api/command/user/iso/AttachIsoCmd.java

```diff
@@ -54,6 +54,9 @@
             required = true, description = "the ID of the virtual machine")
     protected Long virtualMachineId;
 
+    @Parameter(name=ApiConstants.FORCED, type=CommandType.BOOLEAN, description="If true, ejects existing iso before attaching on vmware. Default: false", since = "4.15.1")
+    protected Boolean forced;
```

Comment:
IMO this is not needed. In case there an existing ISO it can be ejected by detachIso setting forced = true

Reply:
That's true @nvazquez. However, this has been done in cases where we have a configdrive iso- under such cases, we can't do a detach operation.

Reply:
Thanks for claryfing @Pearl1594